usaspending_api/common/helpers/business_categories_helper.py

```diff
@@ -0,0 +1,143 @@
+        "field_name": "community_developed_corporation_owned_firm"
+    },
+    {"display_name": "U.S.-Owned Business", "field_name": "us_owned_business"},
+    {
+        "display_name": "Foreign-Owned and U.S.-Incorporated Business",
+        "field_name": "foreign_owned_and_us_located_business"
+    },
+    {"display_name": "Foreign Owned and Located", "field_name": "foreign_owned_and_located_business"},
+    {"display_name": "Foreign Government", "field_name": "foreign_government"},
+    {"display_name": "International Organization", "field_name": "international_organization"},
+    {"display_name": "Domestic Shelter", "field_name": "domestic_shelter"},
+    {"display_name": "Hospital", "field_name": "hospital"},
+    {"display_name": "Veterinary Hospital", "field_name": "veterinary_hospital"},
+
+    # Nonprofit
+    {"display_name": "Nonprofit Organization", "field_name": "nonprofit"},
+    {"display_name": "Foundation", "field_name": "foundation"},
+    {"display_name": "Community Development Corporation", "field_name": "community_development_corporations"},
+
+    # Higher education
+    {"display_name": "Higher Education", "field_name": "higher_education"},
+    {
+        "display_name": "Higher Education (Public)",
+        "field_name": "public_institution_of_higher_education"
+    },
+    {
+        "display_name": "Higher Education (Private)",
+        "field_name": "private_institution_of_higher_education"
+    },
+    {
+        "display_name": "Higher Education (Minority Serving)",
+        "field_name": "minority_serving_institution_of_higher_education"
+    },
+    {"display_name": "Educational Institution", "field_name": "educational_institution"},
+    {"display_name": "School of Forestry", "field_name": "school_of_forestry"},
+    {"display_name": "Veterinary College", "field_name": "veterinary_college"},
+
+    # Government
+    {"display_name": "Government", "field_name": "government"},
+    {"display_name": "U.S. National Government", "field_name": "national_government"},
+    {"display_name": "U.S. Regional/State Government", "field_name": "regional_and_state_government"},
+    {"display_name": "U.S. Regional Government Organization", "field_name": "regional_organization"},
+    {"display_name": "U.S. Interstate Government Entity", "field_name": "interstate_entity"},
+    {"display_name": "U.S. Territory Government", "field_name": "us_territory_or_possession"},
+    {"display_name": "U.S. Local Government", "field_name": "local_government"},
+    {"display_name": "Native American Tribal Government", "field_name": "indian_native_american_tribal_government"},
+    {"display_name": "U.S. Government Authorities", "field_name": "authorities_and_commissions"},
+    {"display_name": "Interstate Entity", "field_name": "interstate_entity"},
+    {"display_name": "Council of Governments", "field_name": "council_of_governments"},
+
+    # Individuals
+    {"display_name": "Individuals", "field_name": "individuals"},
+]
```

Comment:
Have we checked that these strings don't appear elsewhere in the codebase? I would like to see one source of truth for these.

Reply:
I did not see anywhere that had the display name. Seemed that everything used the underscore name only. Even the API Contract says that it will return the display name, but returns the underscore name.

Reply:
Well I would like to see the underscore names, even if not connected to the display name, all come from the same list.

Reply:
Gotcha, I am following now. I would really like to get a complete list of these strings and then go and make a lookup for them. The get_business_categories.py file seemed to be where the strings originated so I used those, but there does seem to be different sets of strings out there.

Reply:
I trust you to take targets of opportunity as you see fit.